icons/jetbrains/jetbrains-plain.svg

```diff
@@ -1 +1,9 @@
HzGxUQWdi0G8lQEtqYpttJIBQAbSBMIEMm1Ew3FHzXAdyiOEPoAVmNmpmRyRNiECBbKnp1W22a55rQU7RqQAioDOBjJsRh4ChC+hUAOugM4Ec4BAwdA2dCmAVMytL2qtgO+2yz1qADlTFNtvoMkIFECJMIEcOOucO+C4CxUOoQOERJpAjcwpWdlQ914GCIlSgsAgTyJlxBR2Kmuc6UGCEChQOYQI5MycOAUNKECpQGGY2qGBZ6A4RJpAPUwoCRc1zHYAkQgUKoC5MjPqtBIhNTWyzjRQiVCC3CBPIqSnRnUBKsfkVcsfMBs1sQtIRESiQHzUFYYJzO5BadCqQG3QmkGMzYpttZACdCmSemQ2b2WHRmUD+1MQ228gQOhXILDMbVtCZGPZbCZCIqthmGxlDpwKZE3YmpiVNi0CBfDronBsiUCBr6FQgM+hMoACqojuBDKNTgdSjM4GCGKc7gayjU4HUMrNRSXeKIIF8mxPbbCMnCBVInTBM7Jc06LcSIHHj4hAw5AihAqlBmECB1MQ228ghQgW8I0ygYKbENtvIKUIFvAlXc0yIMIFiqInuBHKOUIGuC7fTnhATMFEcM2KbbRQAS0rRVWZ2QNKsCBQohprYZhsFQqcCXUF3AgU0I7oTKBg6FUicme0Q3QkUy0G6EygiOhVIlJmNSdrruw6gS6pim20UGJ0KJMLMyuFx5Ht91wJ0CYeAofDoVCB2ZlZWcE5HxW8lQFfMiW22AUl0KhAzAgUKZlzSEIECCNCpQGwIFCiQmqSdhAngXHQqEAsCBQpkStKVBArgfHQq0DECBQqiJrbZBtZFpwJxmBCBAvk2paA7MeW5DiDVCBXoSLgPxQ7fdQAJqSnYZnsnp4oCzTH8gbaFO2Ue9l0HkJAZsc020BJCBdoSnuUxK6nstxIgEfucc+O+iwCyptd3AciswyJQIH+qYpttoG3MqUDLwuPLK57LAOLGNttAhxj+QEvCYY8jvusAYjSnYCOrquc6gMyjU4FWTfguAIjRuIJttque6wBygTkViCxc7THsuQwgDnPiEDAgdgx/IDIzOyJp0HcdQIcmFazuqHmuA8gdOhWIxMxGRaBAttXENttAouhUIBK6FMi4KQWBoua5DiDXmKiJpuhSIMNqCsIE22wDXUCnAk3RpUBGzYhttoGuolOBdZnZsAgUyJaagomYIwQKoLuYqIlm7vRdANCCqoKNrOY81wEUEsMfWJOZlSUd910HENFB59wB30UARUanAuvZ4bsAIIKqOAQMSAXmVGA9t/ouAGhinEPAgPRg+AMNMfSBlJsT22wDqUOnAmsZ9l0AsIZxBYeAzXiuA8AqzKnAWhj6QNrUxDbbQKrRqcBaKr4LAOpMSbqSQAGkG3MqcB7mUyBFaqI7AWQGwx9opOK7AEDBNtuc2QFkCMMfaGTYdwEotJpe3Wa75rkWAC2gU4FGrvBdAAprRhwCBmQWnQo0Mui7ABTSQQ4BA7KNiZo4j5kdl1T2XQcKoyq22QZygU4FGin7LgCFcZBttoH8YE4FzhEuJwWSNie22QZyh04FVqv4LgC5Ny622QZyiU4FgG6ZE90JINfoVADohinRnQByj04FgCTVxDbbQGHQqQCQlClxCBhQKIQKAHGrKdhmm3M7gIJh+ANAnGbENttAYdGpABCHml49BGzOcy0APGGbbpwj3PzquO86kClVsc02ANGpwCqMgaNFbLMN4CzmVKCRmjj/A+uriu4EgFXoVKCRqu8CkGrjkkYIFABWo1OBRuZ8F4BUmhPbbANYB50KNPKc7wKQOpNim20ATRAq0MiM7wKQGjVJO51zu5nEC6AZlpTiPCwrRWhKwXBHzXMdADKCTgXOE15E5jyXAX9qCroTbLMNoCWECqxlxncB8GJGwdyJKc91AMggQgXW8rDvAtBVNbHNNoAOMacCDTGvolCqCoY75jzXASDj6FSgoXAsfcpzGUjeyjbbc74LAZB9hAqs50HfBSAxVQVzJw54rgNAjjD8gTUxBJJb4865fb6LAJA/dCqwJoZAcmdOwZkdBAoAiSBUoJl7fReAWIyLbbYBJIzhDzRlZkckDfquA22pKVjZMeO5DgAFQKcCURz0XQDaMiXpSgIFgG6hU4FIzOy4pLLvOhBJTcGZHVOe6wBQMHQqEBXdimyYUdCdmPJcB4AColOByJhbkWo1BRtZjXuuA0CB0alAK+hWpNOMgpUd457rAFBwdCrQEjObljTsuw6cdZBdMQGkBaECLTGziqRZ33VAVQWTMaue6wCAsxj+QEvCixjDIH6tHAJW9V0IANSjU4G2mNmspIrvOgpmTsFGVlXPdQBAQ3Qq0K7dClYcoDvGFUzGrHquAwDWRKcCbTOzUUkTvuvIuTkFcydmPNcBAE3RqUDbnHOTkiY9l5FnU+IQMAAZQqcCHWN+RexqYpttABlEpwJxGFGwxBGdmxLbbAPIKDoViIWZDSrYv6Lst5LMqknaFw4pAUAmESoQm3BjrGkRLFo1o2C4Y85zHQDQEYY/EJtwueOIWGoaVU1Bd2KEQAEgD+hUIHbhUMhhMXlzPVWxzTaAnKFTgdiF77qZvLk2ttkGkEt0KpAoM5uQNOq7jpSoiu4EgByjU4FEOed2S9rnu44UGJc0QqAAkGd0KtAV4cqQw5IG/VbSdXNim20ABUGnAl0RvkMfUvCOvSjGxTbbAAqETgW6zsyGJY0pv6tD5kR3AgCA7jGzvWZ23PLjuJnt9f26AoAvdCrglZmVJe2VdLeyuxNnTdJnJI0752p+SwEAoODMrGxmByxbnYvjFtRc9v36AQCABsxs1MxmfaaFJmbNbNT36wQAacPwB1LLgmWod0raIf9LUecUHEt+L3tNAEBjhApkQhgwhiXdGv63G2YkPShphiABAM0RKpBJFixLrUjapqCLMdzhQ84o6EY8KanKclAAaB2hArkSho0Vw2vcrKpXj2evsmIDAOLx/wMpKeQ1ZqTHOAAAAABJRU5ErkJggg=="/>
```

Comment:
Looks like you're adding a PNG file in an SVG file. That won't work properly. 😅

Reply:
@JOS-RE: Pl head over to this! :eyes: